x-pack/plugins/observability/public/components/app/section/apm/index.tsx

```diff
@@ -85,7 +86,9 @@ export function APMSection({ bucketSize }: Props) {
   const min = moment.utc(absoluteStart).valueOf();
   const max = moment.utc(absoluteEnd).valueOf();
 
-  const formatter = niceTimeFormatter([min, max]);
+  const formatter = bucketSize?.dateFormat
+    ? timeFormatter(bucketSize?.dateFormat)
+    : niceTimeFormatter([min, max]);
```

Comment:
Why not also use `timeFormatter` here?

Reply:
Do you mean why we are using `niceTimeFormatter` here?